appcenter-data/ios/AppCenterReactNativeData/AppCenterReactNativeData.m

```diff
@@ -48,20 +54,40 @@ @implementation AppCenterReactNativeData
 
 static NSString *const kMSjsonValueKey = @"jsonValue";
 
-static NSString *const kMSPartitionKey = @"partition";
-
-static NSString *const kMSIDKey = @"id";
-
 static NSString *const kMSETagKey = @"eTag";
 
 static NSString *const kMSLastUpdatedDateKey = @"lastUpdatedDate";
 
 static NSString *const kMSIsFromDeviceCacheKey = @"isFromDeviceCache";
 
+static NSString *const kMSIDKey = @"id";
+
+static NSString *const kMSPartitionKey = @"partition";
+
 static NSString *const kMSTimeToLiveKey = @"timeToLive";
 
+static NSString *const kMSErrorKey = @"error";
+
+static NSString *const kMSPaginatedDocumentsIDKey = @"paginatedDocumentsId";
+
+static NSString *const kMSItemsKey = @"items";
+
+static NSString *const kMSCurrentPageKey = @"currentPage";
+
+static NSString *const kMSMessageKey = @"message";
+
 RCT_EXPORT_MODULE();
 
+- (instancetype)init {
+    self = [super init];
```

Comment:
Need to check for `nil`. You can do something like this:

```objective-c
if ((self = [super init])) {
  _paginatedDocuments = [[NSMutableDictionary alloc] init];
}
return self;
```

Reply:
Changed